javascript/ql/src/semmle/javascript/Modules.qll

```diff
@@ -160,6 +160,11 @@ abstract class Import extends ASTNode {
       result = resolveFromTypeRoot()
     )
   }
+
+  /**
+   * Gets the data flow node that the default import of this import is available at.
+   */
+  abstract DataFlow::Node getDefaultNode();
```

Comment:
I think it's great that some of this logic is moved into the modules instead of being a list of special cases in the data flow libraries, but the name "default" is a bit unfortunate, as it seems to be based on the non-standard interpretation of ES6 default imports. How about just `getImportedModuleNode`?